### ISSUE TITLE
我想请问的是您另一个FMDNet网络的问题，看您最近在这个项目有更新，也在这写一个。

### ISSUE THREAD

**Issue by wudan99** [Wenzhuo-Liu/UMD-Net#1]
在您的FAEF模块当中，明确使用了注意力机制Q,K,V，在您的网络图中我们可以看到Q,Ek是有矩阵乘法，以及sofemax之后和Ev也是有乘法（从您的论文公式中也有体现），但是您的代码中对于这块的内容却和您说的并不相符：以下代码是从您的FAEF模块截取的。
                x = torch.cat((aud, img, inc, kfx, kfy, kfz, rol, yaw), dim=1)
		# print(x.shape) # [32, 1024, 1, 1]
		x = self.oneconv512(x)
		x = self.oneconv256(x)
		x = self.oneconv128(x)# [32, 128, 1, 1]

		idn = x # [32, 128, 1, 1]
		x = self.conv1(x)
		b, c, h, w = x.size()
		n = h*w
		x = x.view(b, c, h*w)   # 如果按照您的网络图，这个就是Q
		attn = self.linear_0(x) # Ek
		attn = F.softmax(attn, dim=-1) # 此处您仅仅对Ek进行归一化，并未Q*Ek
		attn = attn / (1e-9 + attn.sum(dim=1, keepdim=True)) # 此处也是有点多此一举
		x = self.linear_1(attn) # 您的意思是此处是Ev
		x = x.view(b, c, h, w) #  对Ev加权去哪了？？
		x = self.conv2(x) 
               。。。。
完全没有其他的意思，我在想是不是您上传代码时哪里忘记更新了，您的这个论文对于我毕业有很重要的作用，希望能得到您的解答，或者是您上传错误，希望能给我这块的正确代码，太感谢了！！！
或者您也可以给我发邮件解答，实在是不胜感激。
3506579284@qq.com
